redwood-treehouse/src/hostMain/kotlin/app/cash/redwood/treehouse/TreehouseLauncher.kt

```diff
@@ -104,10 +86,30 @@ public class TreehouseLauncher internal constructor(
    * Continuously polls for updated code, and emits a new [LoadResult] instance when new code is
    * found.
    */
-  private fun ziplineFlow(spec: TreehouseApp.Spec<*>): Flow<LoadResult> {
+  private fun ziplineFlow(treehouseApp: TreehouseApp<*>): Flow<LoadResult> {
+    val spec = treehouseApp.spec
+
+    // Loads applications from the network only. The cache is neither read nor written.
+    val ziplineLoaderNetworkOnly = ZiplineLoader(
```

Comment:
... and to make that work, I’m not sharing a `ZiplineLoader` between `TreehouseApp` instances.